ts/features/bonus/bonusVacanze/store/reducers/availableBonusesTypes.ts

```diff
@@ -80,6 +80,21 @@ export const visibleAvailableBonusSelector = createSelector(
     )
 );
 
+export const isAvailableBonusLoadingSelector = createSelector(
+  availableBonusTypesSelector,
+  (abs: AvailableBonusTypesState) => pot.isLoading(abs)
+);
+
+export const isAvailableBonusNoneErrorSelector = createSelector(
```

Comment:
In this selector you could use composition 

```typescript
export const isAvailableBonusNoneErrorSelector = createSelector(
  [availableBonusTypesSelector, isAvailableBonusErrorSelector],
  (abs: AvailableBonusTypesState, hasError: boolean) =>
    hasError && pot.isNone(abs)
);
```

PS you could also leave a little comment for each selector?